packages/icons/icons.json

```
@@ -3075,644 +3075,644 @@
+        "tags": "doctor, science, heart, stethoscope, medical, examination, medicine, hospital, treatment",
         "group": "interface",
         "codepoint": 61817
     },
     {
         "displayName": "Lab test",
         "iconName": "lab-test",
-        "tags": "science, beaker, chemistry, mixing, scientist",
+        "tags": "science, beaker, chemistry, mixing, scientist, test tube, flask, formula, drug",
         "group": "interface",
         "codepoint": 61975
     },
     {
         "displayName": "Virus",
         "iconName": "virus",
-        "tags": "virus, doctor, medical, examination, disease, pandemic, sick",
+        "tags": "doctor, medical, examination, disease, pandemic, sick, germ, covid, planet, explosion, black star",
         "group": "interface",
         "codepoint": 62224
     },
     {
         "displayName": "Inherited group membership",
         "iconName": "inherited-group",
-        "tags": "group, inheritance, inherited, membership, people",
+        "tags": "inheritance, membership, people, share",
         "group": "interface",
         "codepoint": 61954
     },
     {
         "displayName": "Hat",
         "iconName": "hat",
-        "tags": "head, clothing, cap, fedora, role, level, permission",
+        "tags": "head, clothing, cap, fedora, role, level, permission, detective, inspector gadget",
         "group": "miscellaneous",
         "codepoint": 61921
     },
     {
         "displayName": "Cycle",
         "iconName": "cycle",
-        "tags": "cycle, bike, cyclist, biker, ride, rider, transportation",
+        "tags": "bike, cyclist, biker, rider, transportation, rossing, pedestrian, bicyclist, cyling, motorcyle",
         "group": "interface",
         "codepoint": 61808
     },
     {
         "displayName": "Route",
         "iconName": "route",
-        "tags": "route, destination, travel, journey, road, transportation",
+        "tags": "destination, travel, journey, road, transportation, directions, maps, navigation",
         "group": "interface",
         "codepoint": 62107
     },
     {
         "displayName": "Modal",
         "iconName": "modal",
-        "tags": "modal, application, layout, user-interface, UI",
+        "tags": "application, layout, user-interface, UI, browser, window, app",
         "group": "interface",
         "codepoint": 62024
     },
     {
         "displayName": "Modal filled",
         "iconName": "modal-filled",
-        "tags": "modal, application, layout, user-interface, UI",
+        "tags": "modal, application, layout, user-interface, UI, window, computer, app",
         "group": "interface",
         "codepoint": 62023
     },
     {
         "displayName": "Drawer left",
         "iconName": "drawer-left",
-        "tags": "drawer, application, layout, user-interface, UI",
+        "tags": "application, layout, user-interface, UI, pass, export, transform, move, sliding door, flow right block",
```

what does "pass" mean in this context?

---

i guess like passing an object to someone? You can remove

---

yeah let's remove `pass`

```suggestion
        "tags": "application, layout, user-interface, UI, export, transform, move, sliding door, flow right block",
```